src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -90,6 +90,15 @@
 						{{ t('spreed','Create a new conversation') }}
 					</NcActionButton>
 
+					<NcActionButton v-if="!hasNoteToSelf"
+						close-after-click
+						@click="restoreNoteToSelfConversation">
```

Comment:
We agreed to not do that, instead the "Note to self" Message action would recreate it.
Now that the code is already here, I wouldn't mind.

Cc @jancborchardt